lib/6to5/to-fast-properties.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * A trick from Bluebird to force V8 to use fast properties for an object.
+ * Read more: http://stackoverflow.com/questions/24987896/
+ *
+ * Use %HasFastProperties(obj) and --allow-natives-syntax to check whether
+ * a particular object already has fast properties.
+ */
+module.exports = function toFastProperties(obj) {
+  /*jshint -W027*/
+  function f() {}
+  f.prototype = obj;
+  return f;
+  eval(obj);
```

Comment:
I'm not really sure why `eval` is needed but seemingly the intention is to avoid any possible optimizations of `toFastProperties` itself or it will break.

http://stackoverflow.com/questions/24987896/how-does-bluebirds-util-tofastproperties-function-make-an-objects-properties#comment38866125_24989927

You can verify the method does its job by doing `console.log(%HasFastProperties(obj))` with `node --allow-natives-syntax`.

Method in Bluebird: https://github.com/petkaantonov/bluebird/blob/master/src/util.js#L212